syft/pkg/cataloger/dotnet/test-fixtures/packages.lock.json

```diff
@@ -0,0 +1,55 @@
+{
+    "version": 1,
+    "dependencies": {
+        "net8.0": {
```

Comment:
I think this is where my understanding goes sideways. 
Is the entry at the top of this map (net8.0) always singular, and where (if anywhere) does the metadata come from that describes `net8.0`

Reply:
As I understand it, `net8.0` is the runtime target version for the packages listed as its siblings. The packages.lock.json file just lists all of these and does not say which one to use like the .deps.json file does.